graphql-dgs-subscriptions-websockets-autoconfigure/build.gradle.kts

```diff
@@ -20,4 +20,6 @@ dependencies {
 
     implementation("org.springframework.boot:spring-boot-autoconfigure")
     implementation("org.springframework:spring-websocket")
+    implementation("jakarta.annotation:jakarta.annotation-api")
```

Comment:
Do we need this new dependency?

Reply:
The project is being built with JDK 1.8, so we don't need this annotation to support `@PostConstruct`. I would argue that we should merge this PR without this dependency, and as a feature ticket, refactor the project such that we move to `"jakarta.annotation:jakarta.annotation-api"`. If you are having trouble building your project in the IDE just make sure it is using JDK 8 for the project.

Reply:
In fact, I do wonder how things are working since it is known that using `PostConstruct` with an explicit dependency to `jakarta.annotation:jakarta.annotation-api` will not work. ref https://github.com/spring-projects/spring-boot/issues/23716

Reply:
I correct myself, we are getting it via the `org.springframework.boot:spring-boot-starter`.

Reply:
removed 'jakarta.annotation:jakarta.annotation-api' and build success! @berngp